AppVeyor CI: openal-soft might be broken

During development of a branch I was unable to get a good appveyor build. Most likely due to #138 and another independent (?) openal-soft error.

```
Starting package 1/5: openal-soft:x64-windows
Building package openal-soft[core]:x64-windows... 
-- CURRENT_INSTALLED_DIR=C:/Tools/vcpkg/installed/x64-windows
-- DOWNLOADS=C:/Tools/vcpkg/downloads
-- CURRENT_PACKAGES_DIR=C:/Tools/vcpkg/packages/openal-soft_x64-windows
-- CURRENT_BUILDTREES_DIR=C:/Tools/vcpkg/buildtrees/openal-soft
-- CURRENT_PORT_DIR=C:/Tools/vcpkg/ports/openal-soft/.
-- Downloading https://github.com/kcat/openal-soft/archive/openal-soft-1.18.2.tar.gz...
-- Downloading https://github.com/kcat/openal-soft/archive/openal-soft-1.18.2.tar.gz... OK
-- Testing integrity of downloaded file...
-- Testing integrity of downloaded file... OK
-- Extracting source C:/Tools/vcpkg/downloads/kcat-openal-soft-openal-soft-1.18.2.tar.gz
-- Extracting done
-- Applying patch C:/Tools/vcpkg/ports/openal-soft/dont-export-symbols-in-static-build.patch
-- Applying patch C:/Tools/vcpkg/ports/openal-soft/dont-export-symbols-in-static-build.patch done
-- Downloading https://github.com/ninja-build/ninja/releases/download/v1.8.2/ninja-win.zip...
-- Downloading https://github.com/ninja-build/ninja/releases/download/v1.8.2/ninja-win.zip... OK
-- Testing integrity of downloaded file...
-- Testing integrity of downloaded file... OK
-- Configuring x64-windows
vcpkg : CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:56 (message):
At line:16 char:3
+   vcpkg install unicorn sdl2 glew openal-soft enet
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (CMake Error at ...e:56 (message)::String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
    Command failed: ninja;-v
    Working Directory: C:/Tools/vcpkg/buildtrees/openal-soft/x64-windows-rel/vcpkg-parallel-configure
    See logs for more information:
      C:\Tools\vcpkg\buildtrees\openal-soft\config-x64-windows-out.log
Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_cmake.cmake:236 (vcpkg_execute_required_process)
  ports/openal-soft/portfile.cmake:23 (vcpkg_configure_cmake)
  scripts/ports.cmake:72 (include)
Error: Building package openal-soft:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: openal-soft:x64-windows
  Vcpkg version: 0.0.107-2018-03-30-4a9feaa2403e59b3267f8bd02ef2fd687ea5661d
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -4,7 +4,7 @@ clone_depth: 10
 cache:
   - C:\tools\vcpkg\installed\
 
-os: Visual Studio 2017
+os: Visual Studio 2017 Preview
 
 environment:
   # Tell MSYS2 to inherit the current directory when starting the shell

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -35,9 +35,6 @@ install:
           # stick to cmake 3.9.6 since on lower versions it could happen that cmake generates a Makefile that links against gcc_eh
           C:\msys64\usr\bin\bash -lc "pacman --noconfirm -U http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-cmake-3.9.6-1-any.pkg.tar.xz 2>&1"
         } else {
-          mkdir C:\tools\vcpkg\ports\unicorn\
-          Start-FileDownload 'https://raw.githubusercontent.com/Microsoft/vcpkg/8766979444d7652b5c170b487ac39391d85b292e/ports/unicorn/CONTROL' -FileName C:\tools\vcpkg\ports\unicorn\CONTROL
-          Start-FileDownload 'https://raw.githubusercontent.com/Microsoft/vcpkg/8766979444d7652b5c170b487ac39391d85b292e/ports/unicorn/portfile.cmake' -FileName C:\tools\vcpkg\ports\unicorn\portfile.cmake
           vcpkg install unicorn sdl2 glew openal-soft enet
         }
 

